src/Cassandra/QueryProtocolOptions.cs

```diff
@@ -89,7 +79,7 @@ public enum QueryFlags
             }
             PagingState = pagingState;
             SerialConsistency = serialConsistency;
-            _timestamp = timestamp;
+            Timestamp = timestamp;
```

Comment:
Due to my comment above, this needs to be changed after creating the new property (either set the field here or the auto property).